.circleci/config.yml

```diff
@@ -215,7 +215,7 @@ jobs:
           at: ~/mattermost/
       - run:
           command: |
-            curl -f -o server.tar.gz https://pr-builds.mattermost.com/mattermost-server/$(echo "${CIRCLE_BRANCH}" | sed 's/pull\//PR-/g')/mattermost-enterprise-linux-amd64.tar.gz || curl -f -o server.tar.gz https://s3.amazonaws.com/releases.mattermost.com/gitlab/bundle/master/mattermost-enterprise-linux-amd64.tar.gz
+            curl -f -o server.tar.gz https://pr-builds.mattermost.com/mattermost-server/$(echo "${CIRCLE_BRANCH}" | sed 's/pull\//PR-/g')/mattermost-enterprise-linux-amd64.tar.gz || curl -f -o server.tar.gz https://s3.amazonaws.com/releases.mattermost.com/gitlab/bundle/release-6.0/mattermost-enterprise-linux-amd64.tar.gz
```

Comment:
Please don't forget to unchange the lock CI changes.

Reply:
@streamer45  Hi, this is causing the Spinwick PRs to use the June 9 release for the server. We are temporarily reverting this in https://github.com/mattermost/mattermost-webapp/pull/8642 (so we can test the change). Let us know if you want us to merge those changes.

Reply:
@pablovelezvidal Apologies. I've forgotten (yet again) to revert them prior to merge. I did it for cloud branch but not on master. If you include them in your PR it would be great or otherwise I can send a separate fix.

Reply:
I can include it in my PR, it should be merged today. Thanks!